server/src-lib/Hasura/Backends/MSSQL/Types/Instances.hs

```diff
@@ -0,0 +1,123 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- | Instances that're slow to compile.
```

Comment:
How is having instances in a different module help with the compile speed?

Reply:
When editing the Types.hs module your editor/ghci can show type errors immediately instead of the 5s~ it takes to compile all the derived instances code.